### ISSUE TITLE
Deprecated Dependencies

### ISSUE THREAD

**Issue by aqalgorepublic** [ConnectyCube/connectycube-flutter-sdk-releases#44]
This dependency is using older versions of dependencies which in return conflict with other dependencies which are up to date.

Please update the dependencies to the latest flutter 2.0 null-safety versions.

**Comment by TatankaConCube** [comment]
We started upgrading our code for supporting 'null-safety' feature, but some of our dependencies not upgraded for supporting this feature so we cant release our lib without them.

**Comment by aytunch** [comment]
@TatankaConCube which dependencies are you talking about?
Maybe we can help them upgrade to null-safety

Right now ConnectyCube not being null safe is a blocker for us. We need to take action

**Comment by aqalgorepublic** [comment]
I was using *flutter_inapp_purchase* package by *dooboolab.com
<http://dooboolab.com>* with *ConnectyCube* but it conflicted on *http*
package and another dependency conflicted with the *intl* package used in
*ConnectyCube.*

**Comment by aytunch** [comment]
@aqalgorepublic both `http` and `intl` packages have null-safe versions. If they are the only conflicts, then Connecty Cube can easily be migrated

**Comment by aqalgorepublic** [comment]
Yes. This is the only issue I have.
But I am not using the flutter_inapp_purchase package anymore. I have
shifted to another package which could be used with ConnectyCube.

**Comment by TatankaConCube** [comment]
I mean next libs:
```
flutter_webrtc      ✗0.5.7   ✗0.5.7      -           ✗0.5.8 
xmpp_stone          ✗0.3.3   ✗0.3.3      -           ✗0.3.3
```

these big libs not updated for 'null-safety' feature support, but they have old dependencies on other libs, so we can't release our lib with the supporting 'null-safety' feature(((

**Comment by aqalgorepublic** [comment]
Thank you for the clarification. Just one more question, how soon is the
update expected?

**Comment by TatankaConCube** [comment]
We plan to finish update our code near to end of this week, but I'm not sure about dependencies(((

**Comment by aytunch** [comment]
I asked webrtc package and they delivered :)
https://github.com/flutter-webrtc/flutter-webrtc/issues/458#issuecomment-804067703

**Comment by aytunch** [comment]
@TatankaConCube web-rtc is null safe on stable fyi
https://github.com/flutter-webrtc/flutter-webrtc/issues/458#issuecomment-811927003

**Comment by TatankaConCube** [comment]
@aqalgorepublic, @aytunch today we published the first version ([2.0.0-nullsafety.1](https://pub.dev/packages/connectycube_sdk/versions/2.0.0-nullsafety.1)) with 'Null Safety' feature support. In this version, we updated all dependencies to the latest versions. Please try it and share your feedback.

**Comment by TatankaConCube** [comment]
no activity for a long time, so close
